planet/cli/data.py

```diff
@@ -295,6 +342,32 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument("item_types", callback=parse_item_types)
+@click.argument('interval',
+                default=None,
+                type=click.Choice(['hour', 'day', 'week', 'month', 'year'],
```

Comment:
instead of defining this list again, import and use [`STATS_INTERVAL`](https://github.com/planetlabs/planet-client-python/blob/f376018877daf845d3666359082475a25baf9f47/planet/clients/data.py#L35)